com.valvesoftware.Steam.json

```diff
@@ -47,6 +32,29 @@
             "PREFIX": "/app"
         }
     },
+    "add-extensions": {
+        "org.freedesktop.Platform.Compat.i386": {
+            "directory": "lib/i386-linux-gnu",
+            "version": "18.08"
+        },
+        "org.freedesktop.Platform.Compat.i386.Debug": {
+            "directory": "lib/debug/lib/i386-linux-gnu",
+            "version": "18.08",
+            "no-autodownload": true
+        },
+        "org.freedesktop.Platform.GL32": {
+            "directory": "lib/i386-linux-gnu/GL",
+            "version": "1.4",
+            "versions": "1.6;1.4",
```

Comment:
I know this is unrelated to your change but in another commit this should be fixed to: `"versions": "18.08;1.4"`

Reply:
Seems like Steam was just updated, I'll wait until someone updates the sha256 sum, before adding new changes to this branch.

Reply:
The versions were set by @valentindavid

Reply:
The new release looks like udev changes only. Let's ignore for now

Reply:
Added https://github.com/flathub/com.valvesoftware.Steam/issues/193 though

Reply:
@TingPing Done.